backend/mora/service/orgunit.py

```diff
@@ -326,8 +342,12 @@ def get_one_orgunit(c, unitid, unit=None,
             rels['tilhoerer'][0]['uuid'],
         )
 
+    elif details is UnitDetails.MINIMAL:
+        pass  # already done
+    elif details is UnitDetails.INTEGRATION:
+        r["integration_data"] = attrs.get("integrationsdata", "")
```

Comment:
eller det her.

Reply:
Her er meningen at man kun henter integrationsdata ud, når man skal bruge dem - udfra devisen, at der faktisk kunne likke 10 k data for hver organisation, og dem vil vi ikke have med op i brugerinterfacet.

Reply:
Det er også fint. Faktisk bør integrationsdata aldrig sendes netop til brugerinterfacet, gætter jeg på.

Det vil sige, man skal kun have dem, når integrationsparametren er sat. Men det min kommentar referer til er, at hvis du sætter `integrationsdata` ligesom du sætter brugervendt nøgle osv., så kan du skrive `attrs['integrationsdata']` i stedet for `attrs.get(...)`

Reply:
Jeg er jo ikke lorekspert, men kan man forestille sig, at der i en post (fra en ældre lora-db) ikke er integrationsdata, men at det så tilføjes til db_structure.py uden at man retter alle poster?
-
Og hvis det ske, ville vi så ikke have en props uden? 
-
Kan det i givet fald ske i fremtiden, når vi ændrer i databasen?
I db_structures.py ser det ud til at det kun er brugervendtnøgle, der er mandatory ifølge metadata.
Betyder det ikke at der måske ikke er integrationsdata?

Reply:
Nej, man kunne fra nu af ikke forestille sig, at MO-versioner med et integrationsdata-felt i egenskaberne kørte op mod en LoRa, som ikke har det. Det giver ikke mening.

Reply:
Vores AAK-integration kan jo heller ikke køre op mod en LoRa, som ikke har vores særlige udvidelsesfelter.

Reply:
Dvs. jeg tænker, at hvis en egenskab ikke er mandatory og mangler, skal det tolkes som, at den er None eller null eller tom eller whatever, ikke som at den slet ikke er der.